nucypher/blockchain/eth/sol/compile.py

```diff
@@ -105,12 +105,13 @@ def compile(self) -> dict:
 
         self.log.info("Compiling with import remappings {}".format(", ".join(remappings)))
 
-        optimization_runs = 10  # TODO: Move..?
+        optimization_runs = 200  # TODO: Move..?
```

Comment:
I just restored how it really was - wrong parameters were in `compile_files` (`optimize` instead of `optimize_runs`), so function worked with default 200 runs,. I decided to change it here otherwise it confusing